src/gpodder/gtkui/main.py

```diff
@@ -1747,7 +1748,11 @@ def save_episodes_as_file(self, episodes):
                     base, extension = os.path.splitext(copy_from)
                     filename = self.build_filename(episode.sync_filename(), extension)
                     copy_to = os.path.join(folder, filename)
-                    shutil.copyfile(copy_from, copy_to)
+                    try:
+                        shutil.copyfile(copy_from, copy_to)
+                    except (OSError, IOError):
+                        copy_to = os.path.join(folder, re.sub(r"[\"*/:<>?\\|]", "_", filename))
```

Comment:
Check that the new copy_to is different than the original one. Otherwise there is no point in trying again.